pkg/util/signals/signals.go

```diff
@@ -18,20 +18,12 @@ package signals
 
 import (
 	"context"
-	"os"
 	"os/signal"
 	"syscall"
 )
 
 // NewSigKillContext returns a Context that cancels when os.Interrupt or os.Kill is received
 func NewSigKillContext() context.Context {
-	ctx, cancel := context.WithCancel(context.Background())
-	c := make(chan os.Signal, 2)
```

Comment:
@markmandel - I noticed that the library function creates a channel with size 1 regardless of how many signals are being tracked: https://cs.opensource.google/go/go/+/refs/tags/go1.19.4:src/os/signal/signal.go;l=284. I assume that since it's part of the std library and the API is designed to handle multiple signals that it is safe (or even correct) to set the channel to size 1 but wanted to know if there was a reason that 5 years ago you set this to size 2: https://github.com/googleforgames/agones/commit/726af15d4d8b3d40435b74ea9bfa41d1607e2881#diff-e7b27b1d977569544a1feedd454f855f61c641a2f35bf9fe75d4bc15c7cefb35

Reply:
Honestly - not that I can remember! $10 says I copied from some other example that copied from another example?

I'd give more confidence to the Go std library than code I wrote 5 years ago 😄